forge-gui/res/cardsfolder/u/unexpected_allies.txt

```diff
@@ -5,5 +5,5 @@ A:SP$ Pump | ValidTgts$ Creature.YouCtrl+nonToken | TgtPrompt$ Select target non
 SVar:TrigEffect:DB$ Pump | ConditionCheckSVar$ X | Defined$ Targeted | KW$ First Strike
 SVar:X:Count$Valid Creature.NotDefinedTargeted+YouCtrl+sharesNameWith Targeted/Plus.Y
 SVar:Y:Count$ValidGraveyard Creature.YouOwn+sharesNameWith Targeted
-DeckHas:Keyword$Double Team|First Strike
+DeckHas:Keyword$DoubleTeam|FirstStrike
```

Comment:
why are the spaces getting removed?

Reply:
At least for first strike, I started my edit from a situation where about half of the mentions in the context of `Deck*:` don't have spaces. I had a quick look over the code and it seemed like that was accounted for first strike, double strike and double team. I take it the look I gave it was too quick?

Reply:
well it's good you spotted it
but in fact it seems like this was a misunderstanding or typo that has spread
it won't work if it's written wrong / without spaces

Reply:
Duly noted. I'll revert and propagate the newer fix where relevant.

Reply:
For Unexpected Allies, should it be "Double team" instead?

Reply:
yea